1-installing-bolt/README.md

```diff
@@ -4,72 +4,175 @@
+1.  Download the Bolt installer package from [https://downloads.puppet.com/windows/puppet5/bolt-x64-latest.msi.](https://downloads.puppet.com/windows/puppet5/bolt-x64-latest.msi)
+2.  Double-click the MSI file and run the installation.
+3.  Run a Bolt command to verify it has installed correctly.  
 
-```powershell
-choco install ruby
-refreshenv
-gem install bolt
-```
+    ```
+    bolt --help
+    ```
 
-You can check everything has been installed correctly by running `bolt --help`.
 
-# Installing Bolt on macOS
+## Install Bolt on Mac OS X
 
-`bolt` is packaged as a Ruby gem, so you will need a Ruby environment installed. It also makes use of gems with native dependencies so you'll need a compiler toolchain. If you don't already have Xcode or similar installed you can do so with the following.
+1.  Download the Bolt installer package for your macOS version. 
 
-```
-xcode-select --install
-```
+    **Tip:** To find the macOS version number on your Mac, go to the Apple \(\) menu in the corner of your screen and choose **About This Mac**.
 
-macOS comes with Ruby already installed so you should be able to install `bolt` using the built-in `gem` tool:
+    -   10.11 \(El Capitan\) [https://downloads.puppet.com/mac/puppet5/10.11/x86\_64/bolt-latest.dmg](https://downloads.puppet.com/mac/puppet5/10.11/x86_64/bolt-latest.dmg)
+    -   10.12 \(Sierra\) [https://downloads.puppet.com/mac/puppet5/10.12/x86\_64/bolt-latest.dmg](https://downloads.puppet.com/mac/puppet5/10.12/x86_64/bolt-latest.dmg)
+    -   10.13 \(High Sierra\) [https://downloads.puppet.com/mac/puppet5/10.13/x86\_64/bolt-latest.dmg](https://downloads.puppet.com/mac/puppet5/10.13/x86_64/bolt-latest.dmg)
+2.  Double-click the `bolt-latest.dmg` file to mount it and then double-click the `bolt-[version]-installer.pkg` to run the installation.
+3.  Run a Bolt command to verify it has installed correctly.  
 
-```
-gem install bolt
-```
+    ```
+    bolt --help
+    ```
 
-You can check everything has been installed correctly by running `bolt --help`.
 
+## Install Bolt from apt repositories on Debian or Ubuntu
 
-# Installing Bolt on Linux
+Packaged versions of Bolt are available for Debian 8 and 9 and Ubuntu 14.04 and 16.04.
 
-`bolt` is packaged as a Ruby gem, so you will need a Ruby environment installed. It also makes use of gems with native dependencies so you'll need a compiler toolchain. The commands to install these dependencies vary based on which flavor of Linux you are running.
+The Puppet repository for the APT package management system is [https://apt.puppet.com](https://apt.puppet.com). Packages are named using the convention `<PLATFORM_VERSION>-release-<VERSION CODE NAME>.deb`. For example, the release package for Puppet 5 Platform on  8 “Jessie” is `puppet5-release-jessie.deb`.
 
-## CentOS 7/RHEL 7
+**Note:** These packages require you to download the Puppet 5 Platform. To install only the Bolt package you can install the packages directly as well.
```

Comment:
Can we link this and the note in Debian/Ubunutu to the doc that has instructions on how to add the puppet5 repos?